README.md

```diff
@@ -82,6 +82,7 @@
 * Necessary information could be found [here](https://api.rubyonrails.org/classes/ActiveRecord/Inheritance.html)
 ##### What is polymorphic associations in Rails? How it is differ from single table inheritance?
 * Necessary information could be found [here](https://medium.freecodecamp.org/single-table-inheritance-vs-polymorphic-associations-in-rails-af3a07a204f2)
+##### What will be printed after running the code [here](https://gist.github.com/engr-hasanuzzaman/9556121c9fd954eb077ac39ed19fe8d1)?
```

Comment:
Hi @engr-hasanuzzaman. It would be great if you can write answer in another gist and provide it here if you know.